pom.xml

```diff
@@ -793,6 +801,27 @@
             <artifactId>slf4j-reload4j</artifactId>
             <version>${slf4j.api.version}</version>
         </dependency>
+        <!-- Opentelementry libraries -->
```

Comment:
It might be better to specify these as requirements in the POMs for `store`/`graph`, rather than for the whole project.

Reply:
Possibly but the intention is that this could be added to any part of the project as we see fit its not specific to any one part.

Reply:
True, although every part which would want to use them will probably have a dependency on graph or store, which would bring them in as transient dependencies.

Reply:
Not sure I quite agree with relying on transients in this case, if a module uses a library directly it should be defined in the `dependencies` section. Both the `graph` and `store` modules currently use this library directly but the only dependency between them is `store` -> `graph, so the dependency would have to be in the `store` module's pom or both. Having it at a project level removes any duplication or need for transient reliance.

Reply:
That makes sense, although currently this parent POM is used for dependencies which are required by all or almost all modules. While this could be used by other modules outside `store`/`graph`, it's not currently. If in future it is used in many places then it certainly would make sense to put them in here.

It's unfortunate that 5 separate dependencies are required as this does magnify the duplication issue.

Reply:
True the 5 different modules isn't ideal but guess its as its quite modulised if it was just one would be easier to just add directly to `graph` and `store` poms

Reply:
Looking at it again, both `graph` and `store` import the new class `OtelUtil.java` from `common-util`, so would it work to put the dependencies in there only?

Reply:
Good point, probably not a bad place to put it as the intention is all span creation goes through that class (so we have control over if its turned on or off) so any modules that implement opentelementry will use the `common-util` module